### ISSUE TITLE
Create a CHS-jsPsych video-config plugin

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1297]
# TL;DR

We need to create a CHS-jsPsych plugin for initializing the camera/microphone, checking the technical set-up, and offering troubleshooting guidance.

# Narrative

As a CHS researcher, it's important to me that my participants can quickly and easily get their camera and microphone set up before they do any recording. I'd like to make sure that any potential technical problems are checked and resolved before the participant starts the study. Participants should be required to check for the presence and quality of the audio and video in their recordings before they're allowed to move on. 

As a CHS family, I'd like some guidance with troubleshooting technical problems with my device/browser and camera/microphone access. 

# Acceptance Criteria

- [x] Tests
- [x] Documentation
- [x] Scoping: decide how similar this plugin needs to be to the content/layout and functionality of the [EFP video-config frame](https://lookit.readthedocs.io/projects/frameplayer/en/latest/components/exp-video-config/doc.html)
- [x] If more than once device is detected, it must include these as options and reload the feed in response to a new selection
- [x] Must display the selected camera's video feed back to the participant
- [x] Must include the option to do a reload test
- [ ] Must incorporate a microphone check
- [x] Should include some troubleshooting text and information about device/browser compatibility (either standard or parameterized) 

# Implementation Notes

jsPsych has plugins for initializing the [camera](https://www.jspsych.org/7.3/plugins/initialize-camera/) and [microphone](https://www.jspsych.org/7.3/plugins/initialize-microphone/), which include drop-down options for device selection. There's also a [`mirror-camera` plugin](https://www.jspsych.org/7.3/plugins/mirror-camera/) for displaying the webcam feed back to the participant.

This is an example of a plugin that might appeal to researchers outside of CHS. We could ask the jsPsych team whether they're interested in adding these features into existing plugins and/or as a new plugin. Another option is to add it to the jsPsych-contrib repository.

**Comment by becky-gilbert** [comment]
Also, we sometimes get reports that the EFP version of this frame is confusing, especially step 2: https://github.com/lookit/ember-lookit-frameplayer/issues/256 So while developing this lookit-jsPsych version, we could consider changes that might make it clearer to participants what exactly they need to do and whether they've "passed" each check.
